src/models/Site.js

```diff
@@ -52,7 +111,40 @@ class Site {
       pcServers: this.pcServers.map(pc => pc.toJSON()),
       vCenterServers: this.vCenterServers.map(vcsa => vcsa.toJSON()),
       aosList: this.aosList.map(aos => aos.toJSON()),
-      hypervisorList: this.hypervisorList.map(hypervisor => hypervisor.toJSON())
+      hypervisorList: this.hypervisorList.map(hypervisor => hypervisor.toJSON()),
+
+      smtp: {
+        address: this.smtp.address,
+        fromAddress: `${fromTemplate}@${fromDomain}`,
+        fromTemplate: fromTemplate,
+        fromDomain: fromDomain,
+        port: this.smtp.port,
+        securityMode: this.smtp.securityMode,
+        // Leave structure there but they are write only in the API
+        credentials: {
+          username: '',
+          password: ''
+        }
+      },
+
+      ldap: {
+        directoryName: this.ldap.directoryName,
+        directoryUrl: this.ldap.directoryUrl,
+        adminRoleGroupMapping: this.ldap.adminRoleGroupMapping,
+        // Leave structure there but they are write only in the API
+        credentials: {
+          username: '',
```

Comment:
should we consider at least passing back username? and/or some kind of marker that username/creds aren't null. We could also do this by helper function for UI purposes. Likely a future TODO though.

Reply:
Created issue #5 so we don't forget about it.